HW 1/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Homework</title>
+</head>
+<body>
+
+    <button id="myBtn"> Try it </button>
+    
+    <button class="myTogBtn"> Toggle Button </button>
+
+    <script>
+
+        let tryButton = document.getElementById("myBtn");
+        const firstLabel = "myBtn";
+        let toggButton = document.getElementById("myTogBtn");
+    
+        function clickHandler(event){
+            console.log("Hey, you just pressed Try it button");
+            alert("Hi, how are you?");
+        }
+
+        function mouseOverHandler(event){

```

Comment:
you're not using event inside your function so you can omit it.

Reply:
Yes teacher